discover/faq.md

```diff
@@ -12,978 +12,157 @@ Hopefully, you'll find the answers here! If not then please don't hesitate to re
-
-A data marketplace allows providers to publish data and buyers to consume data. Unlike centralized data marketplaces, decentralized ones give users more control over their data, algorithms, and analytics by minimizing custodianship and providing transparent and immutable records of every transaction. With features such as Compute-to-Data (C2D), data and algorithms can be ingested into secure Docker containers where escapes are avoided, protecting both the data and algorithms.
-
-</details>
 
-<details>
-
-<summary>What is needed to use a decentralized marketplace?</summary>
-
-Users access decentralized marketplaces via Metamask. Metamask is an applet interface that manages unique IDs, generated and controlled fully by the user. These unique IDs (aka Ethereum address) are used to store digital assets such as cryptocurrency, datatokens, NFTs, and other web3 native assets.
-
-A Metamask wallet can easily be set up as a browser extension by going to your browser’s web store for extensions and searching for “MetaMask”. For additional help setting up your MetaMask wallet, watch our short tutorial video and review these instructions on Ocean’s documentation page.
-
-Once a user has Metamask installed and an Ethereum address, they can register, consume, or stake on datasets on Ocean Market.
-
-</details>
-
-<details>
-
-<summary>Is Ocean considered to be in the file storage category?</summary>
-
-No, OCEAN does not store data. When publishing, you are providing the URL (or specs for other types of storage).
-  
+This issue is similar to what any digital distribution platform faces. For instance, can Netflix prevent individuals from downloading and redistributing their content? Not entirely. They invest significant resources in security, but ultimately, complete prevention is extremely difficult. They mainly focus on making it more challenging for such activities to occur.
 </details>
 
 <details>
+<summary>Is Acentrik from Mercedes Benz built on top of Ocean?</summary>
 
-<summary>How do I price my data?</summary>
-
-Ocean gives you two different options for pricing your data - [fixed price](../developers/contracts/pricing-schemas.md#fixed-pricing) or [free](../developers/contracts/pricing-schemas.md#free-pricing). You need to decide what your dataset is worth and how you want to price it. You can change the price but you can’t change the price format (e.g. from fixed to free).
+3rd party markets such as Gaia-X, BDP and Acentrik use Ocean components to power their marketplace. They will likely use another currency for the exchange of services. If these marketplaces are publicly accessible, indexable and abide by the fee structure set out by Ocean Protocol, transaction fees would be remitted back to the Ocean community. These transaction fees would be allocated according to plan set out [here](https://blog.oceanprotocol.com/ocean-token-model-3e4e7af210f9).
 
 </details>
 
 <details>
+<summary>Is the Ocean Shipyard funded by $OCEAN?</summary>
```

Comment:
Change "Is the Ocean Shipyard funded by $OCEAN?" to

"What is Ocean Shipyard?"

Why: because the current answer doesn't answer the original question. But the original question is kinda useless, and the answer is useful. Therefore, change the question to fit the answer! :)